static/app/gettingStartedDocs/go/fiber.tsx

```diff
@@ -0,0 +1,255 @@
+
+type Params = DocsParams;
+
+const getConfigureSnippet = (params: Params) => `
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/getsentry/sentry-go"
+	sentryfiber "github.com/getsentry/sentry-go/fiber"
+)
+
+// To initialize Sentry's handler, you need to initialize Sentry itself beforehand
+if err := sentry.Init(sentry.ClientOptions{
+  Dsn: "${params.dsn}",
+  // Set TracesSampleRate to 1.0 to capture 100%
+  // of transactions for performance monitoring.
+  // We recommend adjusting this value in production,
+  TracesSampleRate: 1.0,
+}); err != nil {
+  fmt.Printf("Sentry initialization failed: %v\n", err)
+}
+
+// Later in the code
+sentryHandler := sentryfiber.New(sentryfiber.Options{
+  Repanic:         true,
+  WaitForDelivery: true,
+})
+
+enhanceSentryEvent := func(ctx *fiber.Ctx) error {
+	if hub := sentryfiber.GetHubFromContext(ctx); hub != nil {
+		hub.Scope().SetTag("someRandomTag", "maybeYouNeedIt")
+	}
+	return ctx.Next()
+}
+
+app := fiber.New()
+
+app.Use(sentryHandler)
+
+app.All("/foo", enhanceSentryEvent, func(c *fiber.Ctx) error {
+	panic("y tho")
+})
+
+app.All("/", func(ctx *fiber.Ctx) error {
+	if hub := sentryfiber.GetHubFromContext(ctx); hub != nil {
+		hub.WithScope(func(scope *sentry.Scope) {
+			scope.SetExtra("unwantedQuery", "someQueryDataMaybe")
+			hub.CaptureMessage("User provided unwanted query string, but we recovered just fine")
+		})
+	}
+	return ctx.SendStatus(fiber.StatusOK)
+})
+
+if err := app.Listen(":3000"); err != nil {
+	panic(err)
+}`;
+
+const getOptionsSnippet = () => `
+// Repanic configures whether Sentry should repanic after recovery, in most cases it should be set to true,
+// as fiber includes its own Recover middleware that handles http responses.
+Repanic bool
+// WaitForDelivery configures whether you want to block the request before moving forward with the response.
+// Because Fiber's "Recover" handler doesn't restart the application,
+// it's safe to either skip this option or set it to "false".
+WaitForDelivery bool
+// Timeout for the event delivery requests.
+Timeout time.Duration`;
+
+const getUsageSnippet = () => `
+sentryHandler := sentryfiber.New(sentryfiber.Options{
+  Repanic:         true,
+  WaitForDelivery: true,
+})
+
+enhanceSentryEvent := func(ctx *fiber.Ctx) error {
+  if hub := sentryfiber.GetHubFromContext(ctx); hub != nil {
+    hub.Scope().SetTag("someRandomTag", "maybeYouNeedIt")
+  }
+  return ctx.Next()
+}
+
+app := fiber.New()
+
+app.Use(sentryHandler)
+
+app.All("/foo", enhanceSentryEvent, func(c *fiber.Ctx) error {
+  panic("y tho")
+})
+
+app.All("/", func(ctx *fiber.Ctx) error {
+  if hub := sentryfiber.GetHubFromContext(ctx); hub != nil {
+    hub.WithScope(func(scope *sentry.Scope) {
+      scope.SetExtra("unwantedQuery", "someQueryDataMaybe")
+      hub.CaptureMessage("User provided unwanted query string, but we recovered just fine")
+    })
+  }
+  return ctx.SendStatus(fiber.StatusOK)
+})
+
+if err := app.Listen(":3000"); err != nil {
+  panic(err)
+};`
```

Comment:
Linting fails because of this:
```suggestion
};`;
```
you can run `yarn fix` locally to make sure every issue is fixed.